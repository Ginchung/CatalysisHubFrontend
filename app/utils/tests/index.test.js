import { download } from '../index';

describe('download', () => {
  it('should create a download link', () => {
    expect(download('http://localhost/')).toMatchSnapshot();
  });
});
