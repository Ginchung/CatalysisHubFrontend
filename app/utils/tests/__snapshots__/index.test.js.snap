// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`download should create a download link 1`] = `
<a
  download="http://localhost/"
  href="data:text/plain;charset=utf-8,undefined"
  style="display: none;"
/>
`;
